src/components/Mutations/MutationsTableComponent.vue

```diff
@@ -82,41 +141,45 @@ const props = defineProps({
   }
 });
 
-const mutations = ref<MutationTableRow[]>();
+const mutations = ref<FinancialMutation[]>();
 const selectedMutationId = ref<number>(-1); // TODO: Handle the case when this is not changed
```

Comment:
Heb ze zelf neergezet maar wil je deze TODO's nog aanpakken in deze PR?

Reply:
Ik zie persoonlijk niet een state waarin dit niet veranderd kan worden tbh